### DISCUSSION TITLE
IFC conformity (openBIM)

### DISCUSSION THREAD

**Discussion by lalberts** [globallightingdata/gldf#20] [General]
Hello,
is this new format IFC compliant?  (or E-Class) 
is it to be proposed to the IES community? (being then a world standard?)
or whichever the steps will be to get to some worldwide standards.

Thanks
Lukas

**Comment by FBremecker** [comment]
Hi Lukas,
Buildingsmart is aware of the GLDF (and likes it very much) but was not involved in the definition process.
The IFC format is intended to exchange project information (buildings etc.) not product information.
The parameters for lighting products in IFC are all availble in the GLDF format.
A "mapping" of IFC or eclas parameters to and from GLDF should be no problem at all. But of course it needs a procedure or tool to do so.

Regards,
Friedrich
